src/showplanner/Player.tsx

```diff
@@ -56,6 +59,88 @@ function PlayerNumbers({ id }: { id: number }) {
   );
 }
 
+const setTrackIntro = (
+  track: api.Track,
+  secs: number,
+  player: number
+): AppThunk => async (dispatch) => {
+  try {
+    await api.setTrackIntro(track.trackid, secs);
+    dispatch(MixerState.setLoadedItemIntro(player, secs));
```

Comment:
Do we want to have loading states for these? If the server's being slow, we're not getting much feedback.

Reply:
hmm. I'll try

Reply:
I've just made it update the UI first, if it fails, no big deal.